src/Caption.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import constants from './constants';
+
+const Caption = ({ className, placement, children, ...props }) => {
+  return (
+    <div
+      className={cx(
+        'caption',
+        {
+          [`${placement}-align`]: placement
+        },
+        className
```

Comment:
seems like placement is always defined, so this could be like:
```jsx
'caption',
`${placement}-align`,
className
```

Reply:
Nice catch